shaders/src/common_getters.fs

```diff
@@ -43,7 +43,14 @@ highp vec4 getResolution() {
 
 /** @public-api */
 highp vec3 getWorldCameraPosition() {
-    return frameUniforms.cameraPosition;
+    // NOTE: for perspective camera only
+    return frameUniforms.cameraPositionOrForward;
+}
+
+/** @public-api */
+highp vec3 getWorldCameraForward() {
+    // NOTE: for orthographic camera only
+    return frameUniforms.cameraPositionOrForward;
```

Comment:
It is quite confusing that only one of `getWorldCameraPosition()` and `getWorldCameraForward()` works, depending on the projection used.

Instead, I suggest introducing a new function in `common_getters.fs`, which may be used to compute unnormalized view vector, and which works in perspective and ortho as well. Since `shading_view/getWorldViewVector()` already returns the direction, a new e.g. `getWorldViewDistance()` could return the magnitude.

Reply:
I went with introducing `getWorldViewVectorFrom()` (311496e62927a3a9d4753d3c6a31d83eabb3aeb6). For the ortho camera, it is the same as using `getWorldViewVector()` and `getWorldViewDistance()` but for the perspective, there is less calculation. I am up for name suggestions